src/stores/gitClone.js

```diff
@@ -0,0 +1,18 @@
+import { defineStore } from 'pinia';
+
+const simpleGit = require('simple-git');
+
+export const gitCloneStore = defineStore('gitClone', {
+  actions: {
+    cloneRepo: async function (repoUrl, repoPath) {
+      return simpleGit()
+        .clone(repoUrl, repoPath)
+        .then((res) => {
+          console.log(res);
```

Comment:
After the clone succeeds we should add it to the sidebar with `addRepoToList`.

Then we can navigate to the `/commits` route. May need to await this action int the component and then have the component navigate to the page.